frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx

```diff
@@ -49,6 +50,7 @@ const TRUNCATE_WIDTH = 780;
 
 const HEADER_HEIGHT = 36;
 const ROW_HEIGHT = 36;
+const LINE_HEIGHT = 15;
```

Comment:
pretty brittle of course, let me know of ideas computing multiline cell height... the approach here is to count number of line breaks in every cell that is formatted with multiline, pick the max, then multiply by this 😅 

we could of course render then get the element's height...